video_xblock/static/html/brightcove.html

```diff
@@ -1,6 +1,6 @@
 <style>
-.bc-player-{{player_id}}_default {
-    width: 100%;
+.bc-player-{{player_id}}_default, .bc-player-{{player_id}}_default-index-0 {
+    width: 100% !important;
     overflow: visible;
 }
 .bc-player-{{player_id}}_default .vjs-menu-button-inline:hover, .bc-player- {{player_id}}_default.vjs-no-flex .vjs-menu-button-inline {
```

Comment:
@wowkalucky is this a mistake on line 6?  `.bc-player- {{player_id}}_default.vjs-no-flex ` (extra space), instead of `.bc-player-{{player_id}}_default.vjs-no-flex `